pyproject.toml

```
@@ -24,10 +24,11 @@ maintainers =[
 ]
 description = """PECOS is a library/framework for the evaluation, study, and design of QEC protocols. It also provides the ability to study and evaluate the performance advanced hybrid quantum/classical compute execution models for NISQ algorithms and beyond."""
 readme = "README.md"
-requires-python = ">=3.9"
+requires-python = ">=3.10"
 license = { file = "LICENSE"}
 keywords = ["quantum", "QEC", "simulation", "PECOS"]
 dependencies = [
+    "phir~=0.1.3",
```

```suggestion
    "phir~=0.1.4",
```
[Released today](https://github.com/CQCL/phir/blob/main/CHANGELOG.md#changed) includes tuple validation.

---

Sounds good. I am going to tweak the workflow to add common oses to the matrix and make sure optional dependency tests run correctly. I'll add it in with the next push while doing this.